frontend/src/routes/manage/Series/Create.tsx

```diff
@@ -0,0 +1,223 @@
+            }}
+        >
+            {/* Title */}
+            <InputContainer>
+                <TitleLabel htmlFor={titleFieldId} />
+                <Input
+                    id={titleFieldId}
+                    required
+                    error={!!errors.title}
+                    css={{ width: 400, maxWidth: "100%" }}
+                    autoFocus
+                    {...register("title", {
+                        required: t("metadata-form.errors.field-required") as string,
+                    })}
+                />
+                {boxError(errors.title?.message)}
+            </InputContainer>
+
+            {/* Description */}
+            <InputContainer css={{ maxWidth: 750 }}>
+                <label htmlFor={descriptionFieldId}>{t("metadata-form.description")}</label>
+                <TextArea id={descriptionFieldId} {...register("description")} />
+            </InputContainer>
+
+            {/* ACL */}
+            <InputContainer css={{ maxWidth: 900 }}>
+                <h2 css={{
+                    marginTop: 32,
+                    marginBottom: 12,
+                    fontSize: 22,
+                }}>{t("manage.shared.acl.title")}</h2>
+                <Controller
+                    name="acl"
+                    control={control}
+                    render={({ field }) => <AclSelector
+                        userIsRequired
+                        onChange={field.onChange}
+                        acl={field.value}
+                        knownRoles={knownRoles}
+                        permissionLevels={READ_WRITE_ACTIONS}
+                    />}
+                />
+            </InputContainer>
+
+            {/* Submit button */}
+            <div css={{ display: "flex", marginTop: 32 }}>
+                <Button
+                    kind="call-to-action"
+                    disabled={!!commitError || inFlight || !isValid}
+                    onClick={onSubmit}>
+                    {t("manage.my-series.create.title")}
+                </Button>
+                <span css={{
+                    display: "flex",
+                    alignSelf: "center",
+                    marginLeft: 12,
+                    position: "relative",
+                    width: 20,
+                    height: 20,
+                }}>
+                    <Spinner
+                        size={20}
+                        css={{
+                            position: "absolute",
+                            transition: "opacity ease-out 250ms",
+                            opacity: inFlight ? 1 : 0,
+                        }}
+                    />
+                    <LuCircleCheck
+                        size={20}
+                        css={{
+                            position: "absolute",
+                            transition: "opacity ease-in 300ms",
+                            opacity: success && !isDirty ? 1 : 0,
+                        }}
+                    />
+                </span>
```

Comment:
This whole "button plus spinner and check" ... I feel like we have that in other places as well. Maybe possible to deduplicate some code?